resources/waf_rules.go

```diff
@@ -75,7 +78,7 @@ func WafRules() *schema.Table {
 					},
```

Comment:
maybe  `data_id` should be  renamed  to `id`.

Reply:
In relations I opted to not change. We can discuss this further